gazebo_ros/test/test_sim_time.cpp

```diff
@@ -62,7 +62,7 @@ TEST_F(TestSimTime, TestClock)
   std::vector<ClockMsg::SharedPtr> msgs;
   std::vector<rclcpp::Time> times;
   auto sub = node->create_subscription<ClockMsg>(
-    "/clock", rclcpp::SystemDefaultsQoS(),
+    "/clock", rclcpp::ClockQoS(),
```

Comment:
Maybe we should also update the publisher QoS to use the same QoS:

https://github.com/ros-simulation/gazebo_ros_pkgs/blob/9954bb9f5c02e463e1227aa3f756247e8c6756df/gazebo_ros/src/gazebo_ros_init.cpp#L163-L165